controllers/user.controller.js

```diff
@@ -225,7 +226,7 @@ export const createAdmin = async (req, res) => {
 };
 export const deleteUser = async (req, res) => {
     const user = res.locals.user;
-    const id = req.body.id;
+    const id = req.params.id;
     try {
         if (user.admin) {
             await POI.deleteMany({user: id})
```

Comment:
This should not be delete many, I don't think it could cause issues, but it should be the correct method to delete a single User

Reply:
Is this not a correct method ?  This will delete all pois related to that  user before deleting the user